### PR TITLE
Fix delete empty collections

### LINKED ISSUE #6: Delete collections that have no members

### PR DESCRIPTION
**Pull request by Timbozand**
Fixes #6.

### CODE REVIEW

**Review by smeets — CHANGES_REQUESTED** [review]
Make a compound query instead of two separate (see removeEntry route)

**Comment by Timbozand** [comment]
Need to delete entries with no relations.

**Comment by Timbozand** [comment]
Fixed so entries with no collection gets deleted

File: src/main/java/se/lth/cs/connect/routes/Collection.java [hunk 1]
```diff
@@ -332,4 +339,16 @@ public RetVal(int mem, int ent) {
         });
 
     }
+    
+    private void removeEntriesWithNoCollection(RouteContext rc){
```
Comment (smeets) [inline src/main/java/se/lth/cs/connect/routes/Collection.java:343]: No need to provide a `RouteContext`?

**Review by smeets — CHANGES_REQUESTED** [review]

File: src/main/java/se/lth/cs/connect/routes/Collection.java (hunk 1 above)
Comment (smeets) [reply src/main/java/se/lth/cs/connect/routes/Collection.java:343]: Need it to access `rc.getLocal("db")` to perform queries.

**Review by smeets — APPROVED** [review]

**Pull request merged**